index.html

```diff
@@ -0,0 +1,131 @@
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap" 
+    rel="stylesheet">
+    <title>Riyaz's Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="site-header">
+        <div id="title-text"><h3>
+          <a class="logo-text" href="#">Riyaz Ahmed</a>
+        </h3>
+      </div>
+        <nav id="navbar">
+          <a class="nav-links" href="#">Portfolio</a>
+          <a class="nav-links" href="#">About</a>
+          <a class="nav-links" href="#">Contact</a>
+          <a><i id="menu" class="material-icons">mail</i></a>
+        </nav>
+        <a href="javascript:void(0);" class="icon">
+          <i class="fa fa-bars"></i>
+        </a>
+      </header>
+
+      <section id="home-content">
+        <div class="left-content">
+            <ul class="desktop-version-list">
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/github.svg" alt="github" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/linkedin.svg" alt="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/angelist.svg" alt="angelist" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/twitter.svg" alt="twitter" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" href="#">
+              <img class="img-icons" src="images/medium.svg" alt="medium" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="main-content">
+          <h1 class="name-text">Hey There. <br>
+             I’m Riyaz</h1>
+          <span class="job-title">I’am a Software Developer</span>
+          <p class="job-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div id="social-icons">
+            <ul>
+              <li class="list1"><a class="social-link" href="#">
+                <img class="img-icons" src="images/github.svg" alt="github" />
+              </a>
+              </li>
+              <li class="list1">
+                <a class="social-link" href="#">
+                <img
+                  class="img-icons"
+                  src="images/linkedin.svg"
+                  alt="linkedin"
+                />
+              </a>
+              </li>
```

Comment:
Your icons have a border that is not present in the template design.

Your icons:

<img width="106" alt="Screen Shot 2022-05-26 at 8 42 42" src="https://user-images.githubusercontent.com/82426682/170499700-262ffaa6-2d39-42af-852b-f6edca064a89.png">

The template design icons:

<img width="157" alt="Screen Shot 2022-05-26 at 8 43 10" src="https://user-images.githubusercontent.com/82426682/170499785-00eb47d7-169e-4fb9-80fb-cb0b497f6f38.png">

Kindly fix this.

Reply:
fixed borders in te design as suggested